src/safeposix/syscalls/fs_calls.rs

```diff
@@ -34,6 +35,8 @@ static LIND_ROOT: &str = "/home/lind/lind_project/src/safeposix-rust/tmp";
 
 const FDKIND_KERNEL: u32 = 0;
 const FDKIND_IMPIPE: u32 = 1;
+const S_FILETYPEFLAGS: i32 = 0o170000;
```

Comment:
We shouldnt define constants here. Is there a place where they are defined @Yaxuan-w ?

Reply:
No, this should be defined inside `src/safeposix/syscalls/fs_constants.rs`